_inc/client/apps/index.jsx

```diff
@@ -147,7 +147,7 @@ const Apps = ( props ) => {
 						{ __( 'Get WordPress apps for any screen.' ) }
 					</p>
 
-					<img src={ imagePath + '/apps/triple-devices.svg' } className="jp-landing-apps__devices" />
+					<img src={ imagePath + '/apps/triple-devices.svg' } className="jp-landing-apps__devices" role="img" alt="Example of three devices to use the WordPress apps. An iPhone, Android phone, and a apple laptop computer." />
```

Comment:
Make alt attribute available for translation like `alt={ __( 'string to translate' ) }`